internal/metrics/prometheus.go

```diff
@@ -129,20 +161,43 @@ func NewCtrlFuncMetrics() *CtrlFuncMetrics {
 		[]string{SuccessKey, ProtocolKey, DataplaneKey},
 	)
 
+	controllerMetrics.ConfigPushSuccessTime = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: MetricNameConfigPushSuccessTime,
+			Help: fmt.Sprintf("The time of the last successful configuration push. "+
+				"`%s` describes the dataplane that was the target of the configuration push.",
+				DataplaneKey,
+			),
+		},
+		[]string{DataplaneKey},
+	)
+
 	if ok := metrics.Registry.Unregister(controllerMetrics.ConfigPushCount); ok {
 		fmt.Printf("Unregistered %s\n", MetricNameConfigPushCount)
```

Comment:
If we want to tell the message in pod logs, would it be better to log them in our structured loggers instead using `fmt.Printf`?

Reply:
Per the other comment, these were never of much use and have been destroyed.